### PR TITLE
Add prop to pass the selected value to the parent for the Select 

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Add prop to pass the selected value to the parent for the Select component

## QA 

- Go to `/?path=/docs/select--select` and see the new prop of the `Select` component
- See the state of the parent component is updated with the `value` of the selected option when the selection is changed

### CODE REVIEW

File: src/components/Select/Select.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ const Select = ({
   help,
   id,
   label,
+  onSelectionChange,
```
Comment (squidsoup) [inline src/components/Select/Select.js:21]: this is typically just called `onChange`

File: src/components/Select/Select.js [hunk 2]
```diff
@@ -54,6 +56,11 @@ Select.propTypes = {
   help: PropTypes.string,
   id: PropTypes.string,
   label: PropTypes.string,
+  /**
+   * Optional function that is called when the selection is changed.
+   * The function gets the selected option as an argument.
+   */
+  onSelectionChange: PropTypes.func,
```
Comment (squidsoup) [inline src/components/Select/Select.js:63]: This comment is a bit redundant, propTypes that are `required` are explicitly marked as such.

**Comment by solazio** [comment]
@squidsoup I will add the test and change the name. Should I remove the whole comment, or only the “optional” part? Thanks

**Comment by squidsoup** [comment]
> @squidsoup I will add the test and change the name. Should I remove the whole comment, or only the “optional” part? Thanks

Cool thanks ovi. Up to you wrt the comment, but I think everyone should already know what an `onChange` event is, well hopefully :)

**Comment by solazio** [comment]
@squidsoup can you please have another look at this

File: src/components/Select/Select.js [hunk 3]
```diff
@@ -34,13 +35,12 @@ const Select = ({
...
       stacked={stacked}
-      success={success}
-    >
+      success={success}>
       <select
         className={classNames("p-form-validation__input", className)}
         id={id}
-        {...props}
-      >
+        onChange={evt => onChange(evt.target.value)}
```
Comment (Lukewh) [inline src/components/Select/Select.js:42]: can we just pass the evt through. also the onChange is optional, so we should only call it if it's defined.

```suggestion
        onChange={evt => onChange && onChange(evt)}
```

File: src/components/Select/Select.test.js [hunk 4]
```diff
@@ -7,18 +7,34 @@ describe("Select ", () => {
   it("renders", () => {
     const wrapper = shallow(
       <Select
-        id="test-id"
+        id='test-id'
```
Comment (Lukewh) [inline src/components/Select/Select.test.js:10]: single quotes!? You having a giggle?

**Pull request merged**